bittensor/mock/subtensor_mock.py

```diff
@@ -1160,7 +1160,7 @@ def do_burned_register(
 
         return True, None
 
-    def _do_stake(
+    def do_stake(
```

Comment:
Will this need to be `pytest.mark.asyncio` parameterised?

Reply:
Yes, but not in this PR